test/rest.test.js

```diff
@@ -2206,7 +2206,7 @@ describe('strong-remoting-rest', function() {
 
```

Comment:
This whole method could be a lot more clear. Why can't we init `hookContext` as an object and just set properties?